playbooks/ci-full/tasks/create.retry

```diff
@@ -0,0 +1 @@
+localhost
```

Comment:
Derp no idea why the retry files crept in. I was being so good about interactive additions to the commit.